src/generators.js

```diff
@@ -1,24 +1,25 @@
-import AdminOnRestGenerator from './generators/AdminOnRestGenerator';
-import ReactGenerator from './generators/ReactGenerator';
-import ReactNativeGenerator from './generators/ReactNativeGenerator';
-import TypescriptInterfaceGenerator from './generators/TypescriptInterfaceGenerator';
-import VueGenerator from './generators/VueGenerator';
+import AdminOnRestGenerator from "./generators/AdminOnRestGenerator";
```

Comment:
I personally disagree with double quotes I find simple quotes easier to read.

Reply:
Me too, but... I prefer to not bother with that and use Prettier's default config.